Containers/apache/start.sh

```diff
@@ -48,7 +48,7 @@ echo "$CADDYFILE" > /Caddyfile
 # Add caddy path
 mkdir -p /mnt/data/caddy/
 
-# Fix apache sturtup
-rm -f /var/run/apache2/apache2.pid
+# Fix apache startup
+rm -f logs/httpd.pid
```

Comment:
Can we use an absloute path here?

Reply:
only there or also on all other locations?

Reply:
for now only here I'd say

Reply:
```suggestion
rm -f /usr/local/apache2/logs/httpd.pid
```